amplipi/updater/asgi.py

```diff
@@ -291,6 +291,38 @@ def install(creds: HTTPBasicCredentials = Depends(security)):
   t.start()
   return {}
 
+class PasswordInput(BaseModel):
+  password: str
+
+@app.post('/password')
+def set_admin_password(input: PasswordInput, creds: HTTPBasicCredentials = Depends(security)):
+  """ Sets the admin password """
+  if len(input.password) == 0:
+    del identity['admin_password_hash']
+  else:
+    # The below takes ~200ms to hash on a RPi3 CM and is currently (fall 2023) recommended
+    # as a 'safer' default by OWASP
+    argon2_params = Argon2Params(
+      time_cost=2,
```

Comment:
if literally every request on the API now has to go through this (because we're using HTTPBasic auth, instead of token bearer), this probably ought to be 1 instead. adding 200ms for every request is absolutely something a user can feel, in terms of response times.

Reply:
What level of effort is using token bearer? Are we planning on implementing it soon?

Reply:
The level of effort is about 2x as much as this, on top of this PR. It's accomplish-able, but I don't personally want to create and manage a login workflow, which is what that would entail; HTTP Basic shifts that responsibility to the browser.